app/frontend/tests/unit/store/modules/form.actions.spec.js

```diff
@@ -88,7 +88,11 @@ describe('form actions', () => {
       await store.actions.fetchDrafts(mockStore, 'dId');
 
       expect(mockStore.commit).toHaveBeenCalledTimes(1);
-      expect(mockStore.commit).toHaveBeenCalledWith('SET_DRAFTS', expect.any(Array));
+      expect(mockStore.commit).toHaveBeenCalledWith('SET_DRAFTS', expect.not.arrayContaining([
+        expect.objectContaining({
+          schema: {}
+        })
+      ]));
```

Comment:
Don't think the test needs to know about `schema` (probably no FE changes needed now)